gravitee-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/UserServiceImpl.java

```diff
@@ -524,6 +525,10 @@ public UserEntity create(NewExternalUserEntity newExternalUserEntity, boolean ad
 
     private UserEntity create(NewExternalUserEntity newExternalUserEntity, boolean addDefaultRole, boolean autoRegistrationEnabled) {
         try {
+            /*
+             TODO: getCurrentEnvironnemenet and call database to fetch the corresponding organization OR add parameters in methods
```

Comment:
real TODO or you forgot to delete it ? :)

Reply:
a TODO not related to this feature